docs/dev/table/hive/hive_catalog.md

```diff
@@ -0,0 +1,245 @@
+   </property>
+
+   <property>
+       <name>hive.metastore.uris</name>
+       <value>thrift://localhost:9083</value>
+       <description>IP address (or fully-qualified domain name) and port of the metastore host</description>
+   </property>
+
+   <property>
+       <name>hive.metastore.schema.verification</name>
+       <value>true</value>
+   </property>
+
+</configuration>
+{% endhighlight %}
+
+
+Test connection to the HMS with Hive Cli. Running some commands, we can see we have a database named `default` and there's no table in it.
+
+
+{% highlight bash %}
+
+hive> show databases;
+OK
+default
+Time taken: 0.032 seconds, Fetched: 1 row(s)
+
+hive> show tables;
+OK
+Time taken: 0.028 seconds, Fetched: 0 row(s)
+{% endhighlight %}
+
+
+#### step 2: configure Flink cluster and SQL CLI
+
+Add all Hive dependencies to `/lib` dir in Flink distribution, and modify SQL CLI's yaml config file `sql-cli-defaults.yaml` as following:
+
+{% highlight yaml %}
+
+execution:
+    planner: blink
+    type: streaming
+    ...
+    current-catalog: myhive  # set the HiveCatalog as the current catalog of the session
+    current-database: mydatabase
+    
+catalogs:
+   - name: myhive
+     type: hive
+     hive-conf-dir: /opt/hive-conf  # contains hive-site.xml
+     hive-version: 2.3.4
+{% endhighlight %}
+
+
+#### step 3: set up a Kafka cluster
+
+Bootstrap a local Kafka 2.3.0 cluster with a topic named "test", and produce some simple data to the topic as tuple of name and age.
+
+{% highlight bash %}
+
+localhost$ bin/kafka-console-producer.sh --broker-list localhost:9092 --topic test
+>tom,15
+>john,21
+
+{% endhighlight %}
+
+
+These message can be seen by starting a Kafka console consumer.
+
+{% highlight bash %}
+localhost$ bin/kafka-console-consumer.sh --bootstrap-server localhost:9092 --topic test --from-beginning
+
+tom,15
+john,21
+
+{% endhighlight %}
+
+
+#### step 4: start SQL CLI, and create a Kafka table with Flink SQL DDL
+
+Start Flink SQL CLI, create a simple Kafka 2.3.0 table via DDL, and verify its schema.
+
+{% highlight bash %}
+
+Flink SQL> CREATE TABLE mykafka (name String, age Int) WITH (
+   'connector.type' = 'kafka',
+   'connector.version' = 'universal',
+   'connector.topic' = 'test',
+   'connector.properties.zookeeper.connect' = 'localhost:2181',
+   'connector.properties.bootstrap.servers' = 'localhost:9092',
+   'format.type' = 'csv',
+   'update-mode' = 'append'
+);
+[INFO] Table has been created.
+
+Flink SQL> DESCRIBE test;
+root
+ |-- name: STRING
+ |-- age: INT
+
+{% endhighlight %}
+
+Verify the table from Hive side via Hive Cli:
+
+{% highlight bash %}
+hive> show tables;
+OK
+test
+Time taken: 0.038 seconds, Fetched: 1 row(s)
+
+hive> describe test;
+OK
+name                	string
+age                 	int
+Time taken: 0.056 seconds, Fetched: 2 row(s)
+
+{% endhighlight %}
+
+
+#### step 5: run Flink SQL to query the Kakfa table
+
+Run a simple select query from Flink SQL CLI
```

Comment:
Mention that SQL CLI needs a Flink cluster to run the query, either standalone or yarn-session?